### PR TITLE
Schema change for NewOrder and Order

### PR DESCRIPTION
**Pull request by udonyx**

### CODE REVIEW

File: protos/otc/v1/responses.proto [hunk 1]
```diff
@@ -9,27 +9,30 @@ import "google/protobuf/timestamp.proto";
 
 
 
-message Order {
+message OtcOrder {
```
[the remark this answers is not in the document]
Comment (udsamani) [reply protos/otc/v1/responses.proto]: Done

File: protos/otc/v1/requests.proto [hunk 2]
```diff
@@ -30,21 +25,23 @@ message Auth {
...
 message Unsubscribe {
-  Channel channel = 1;
+  OtcChannel channel = 1;
   string symbol = 2;
 }
 
 message OtcRequest {
   string id = 1;
   google.protobuf.Timestamp timestamp = 2;
+  OtcRequestMethod method = 3;
```
[the remark this answers is not in the document]
Comment (udsamani) [reply protos/otc/v1/requests.proto]: Done

File: protos/otc/v1/requests.proto [hunk 3]
```diff
@@ -30,21 +25,23 @@ message Auth {
...
 
 message Subscribe {
-  Channel channel = 1;
+  OtcChannel channel = 1;
   string symbol = 2;
 }
 
 message Unsubscribe {
-  Channel channel = 1;
+  OtcChannel channel = 1;
```
[the remark this answers is not in the document]
Comment (udsamani) [reply protos/otc/v1/requests.proto]: Done

File: protos/otc/v1/requests.proto [hunk 4]
```diff
@@ -30,21 +25,23 @@ message Auth {
 }
 
 message Subscribe {
-  Channel channel = 1;
+  OtcChannel channel = 1;
```
[the remark this answers is not in the document]
Comment (udsamani) [reply protos/otc/v1/requests.proto]: Done

File: protos/otc/v1/requests.proto [hunk 5]
```diff
@@ -8,19 +8,14 @@ import "otc/v1/types.proto";
...
+message OtcOrderRequest {
+  optional string client_order_id = 1;
   string account_id = 2;
-  string symbol = 5;
+  OtcTradableSymbol symbol = 5;
   Decimal quantity = 11;
-  Side side = 12;
-  OrderType order_type = 13;
+  OtcSide side = 12;
+  OtcOrderType order_type = 13;
```
[the remark this answers is not in the document]
Comment (udsamani) [reply protos/otc/v1/requests.proto]: Done

File: protos/otc/v1/requests.proto [hunk 6]
```diff
@@ -8,19 +8,14 @@ import "otc/v1/types.proto";
...
-  string client_order_id = 1;
+message OtcOrderRequest {
+  optional string client_order_id = 1;
   string account_id = 2;
-  string symbol = 5;
+  OtcTradableSymbol symbol = 5;
   Decimal quantity = 11;
-  Side side = 12;
-  OrderType order_type = 13;
+  OtcSide side = 12;
```
[the remark this answers is not in the document]
Comment (udsamani) [reply protos/otc/v1/requests.proto]: Done

File: protos/otc/v1/types.proto [hunk 7]
```diff
@@ -53,26 +55,26 @@ enum OrderState {
...
 
+enum TradableSymbol {
+  TRADABLE_SYMBOL_UNSPECIFIED = 0;
+  TRADABLE_SYMBOL_SWAP = 1;
+  TRADABLE_SYMBOL_BUTTERFLY = 2;
+  TRADABLE_SYMBOL_SPREAD = 3;
+}
+
 
 enum OrderActionType {
```
Comment (udsamani) [inline protos/otc/v1/types.proto:66]: @lsbardel currently our model contains action types for updates too should they also be  included. 

```
pub enum OrderActionType {
    Unspecified = 0,
    PendingCreated = 11,
    PendingUpdate = 12,
    PendingCancel = 13,
    Created = 21,
    UpdateCreated = 22,
    CancelCreated = 23,
    Rejected = 31,
    UpdateRejected = 32,
    CancelRejected = 33,
    Fill = 41,
    FillCorrect = 42,
    FillCancel = 43,
    Expired = 51,
    Suspend = 52,
    Disconnect = 53,
    Update = 54,
}
```

Should we include all ?

Comment (udsamani) [reply protos/otc/v1/types.proto]: As discussed offline getting rid of OrderActionType as we no longer need it.

**Review by lsbardel — APPROVED** [review]

**Pull request merged**